index.js

```diff
@@ -82,6 +82,12 @@ async function register (server, options) {
     const childBindings = getChildBindings(request)
     request.logger = logger.child(childBindings)
 
+    if (options.logRequestStart) {
```

Comment:
can you extract this in an outer variable?

Reply:
Done